app/views/admin/users/widgets/_organizations.html.erb

```diff
@@ -0,0 +1,67 @@
+    <%= crayons_icon_tag(:team, title: "Organizations", class: "mr-2") %>
+    Organizations
+  </h2>
+  <p class="color-secondary">Assign a roles to this member based on permissions. <a href="#" class="c-link c-link--branded">Learn more</a></p>
+</header>
+<% if @organization_memberships.load.empty? %>
+  <div class="flex justify-between items-center gap-4">
+    <p>This user hasn't been added to any organization yet.</p>
+    <button class="c-btn c-btn--secondary whitespace-nowrap" type="button" data-modal-trigger data-modal-title="Add organization" data-modal-size="m" data-modal-content-selector="#add-organization">Add to organization</button>
+  </div>
+<% else %>
+  <% @organization_memberships.each do |org_membership| %>
+    <div class="c-card c-card--interactive flex p-2 relative items-center mb-2">
+      <div class="flex items-center gap-2 flex-1">
+        <a href="<%= org_membership.organization.path %>" target="_blank" rel="noopener" class="c-link">
+          <img src="<%= org_membership.organization.profile_image_url_for(length: 64) %>" width="32" height="32" alt="<%= org_membership.organization.name %> logo" class="radius-default align-middle">
+        </a>
+        <h3 class="crayons-subtitle-3">
+          <a href="<%= org_membership.organization.path %>" target="_blank" rel="noopener" class="c-link">
+            <%= org_membership.organization.name %>
+          </a>
+          <span class="color-secondary fw-normal">– <%= org_membership.type_of_user.capitalize %></span>
+        </h3>
+      </div>
+      <div class="c-card__revealable">
+        <button type="button" class="c-btn c-btn--icon-alone" data-modal-trigger data-modal-title="Edit organization" data-modal-size="m" data-modal-content-selector="#edit-organization-<%= org_membership.organization.id %>">
+          <%= crayons_icon_tag(:edit, class: "c-btn__icon") %>
+        </button>
+        <%= form_with model: [:admin, org_membership], method: :delete, local: true, html: { class: "inline" } do |f| %>
+          <%= f.button crayons_icon_tag(:x, class: "c-btn__icon"), class: "c-btn c-btn--destructive c-btn--icon-alone", data: { confirm: "Are you sure you want to remove them from #{org_membership.organization.name}?" } %>
+        <% end %>
+      </div>
+    </div>
+
+    <div id="edit-organization-<%= org_membership.organization.id %>" class="hidden">
+      <%= form_with model: [:admin, org_membership], method: :patch, local: true do |f| %>
+        <div class="crayons-field mb-4">
+          <%= f.label "Permission level", class: "crayons-field__label" %>
+          <p class="crayons-field__description">You can assign user to be admin or regular member.</p>
+          <%= f.select("type_of_user", options_for_select(%w[member admin], selected: org_membership.type_of_user), {}, class: "crayons-select") %>
+        </div>
+        <%= f.button "Update", class: "c-btn c-btn--primary", type: "submit" %>
+      <% end %>
+    </div>
```

Comment:
@nickytonline this relies on current implementation of Modals here and it just feels a bit dirty to me but maybe I'm biased :D 

Also clickable labels do not work here for the reasoning mentioned [here](https://forem-team.slack.com/archives/C01S4QZUDNC/p1643059938100200). I haven't even bothered adding `for` attribute here for that reason as it needs to be figured out first.

Issue is that modals clone these elements (entire forms) to display them as innerHTML but they do not remove the original one. So we end up with having at least TWO form elements having exactly the same ID (which by definition is wrong because IDs should be unique) hence the clickable labels do not work.

Same goes for every modal with form.